.drone/drone.yml

```diff
@@ -0,0 +1,142 @@
+---
+kind: pipeline
+name: Lint
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: lint
+  image: golangci/golangci-lint:v1.37.1
+  commands:
+  - apt-get update -y && apt-get install -y libsystemd-dev
+  - make lint
+---
+kind: pipeline
+type: docker
+name: Dist
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: distribute
+  image: rfratto/seego
+  commands:
+  - apt-get update &&  apt-get install -y rubygems rpm nsis
+  - gem install --no-document fpm
+  - mkdir -p /usr/local/go/bin
+  - wget -q https://golang.org/dl/go1.16.5.linux-amd64.tar.gz
+  - tar -C /usr/local -xzf go1.16.5.linux-amd64.tar.gz
+  - rm go1.16.5.linux-amd64.tar.gz
+  - export PATH=$PATH:/usr/local/go/bin
+  - make DRONE=true BUILD_IN_CONTAINER=false RELEASE_TAG=v0.0.0 dist
+---
+kind: pipeline
+type: docker
+name: Test
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: distribute
+  image: rfratto/seego
+  commands:
+  - apt-get update &&  apt-get install -y rubygems rpm nsis  apt-transport-https ca-certificates curl gnupg lsb-release
+  - gem install --no-document fpm
+  - mkdir -p /usr/local/go/bin
+  - wget -q https://golang.org/dl/go1.16.5.linux-amd64.tar.gz
+  - tar -C /usr/local -xzf go1.16.5.linux-amd64.tar.gz
+  - rm go1.16.5.linux-amd64.tar.gz
+  - export PATH=$PATH:/usr/local/go/bin
```

Comment:
Is there a way we can dedupe some of this stuff? Does Drone support Yaml anchors? If there's no easy/sane way to dedupe, I'm fine with leaving it as-is for now and then generating the drone.yml later on. (Generating with Jsonnet is what Loki did when moving to Drone).

Reply:
I tried using the anchors but could not get them to be happy. From light reading on drone issues, they have changed the behavior a few times.

Reply:
Alright, no worries, a config language can clean this up down the line.